WordPress/src/main/res/values-night/colors.xml

```diff
@@ -101,5 +101,5 @@
     <color name="site_creation_intent_item_emoji_bg">#3d3d3d</color>
 
     <!-- Jetpack Migration -->
-    <color name="bg_jp_migration_buttons_panel">@color/black_translucent_90</color>
+    <color name="bg_jp_migration_buttons_panel">#E6121212</color>
```

Comment:
I added this enhancement so the background of the buttons container is not different on the welcome step vs the other steps.